pkg/controller/performanceprofile/components/profile/profile.go

```diff
@@ -54,3 +54,16 @@ func IsPaused(profile *performancev2.PerformanceProfile) bool {
 
 	return false
 }
+
+// IsRpsEnabled or not RPS mask should be set for all physical net devices
+func IsRpsEnabled(profile *performancev2.PerformanceProfile) bool {
+	if profile.Annotations == nil {
+		return false
+	}
+	isRpsEnabled, ok := profile.Annotations[performancev2.PerformanceProfileEnableRpsAnnotation]
+	if ok && isRpsEnabled == "true" {
```

Comment:
we didn't use `strconv.ParseBool` in the end, didn't we?

Reply:
at the end we did'nt.  Its using the same annotation logic we have in the function above (isPaused)

Reply:
too bad, but no place to fix this in a backport.